### PR TITLE
MUMUP-2054 : Add in rss widget type, add in example in mock data

### PR DESCRIPTION
**Pull request by timlevett**
image

### CODE REVIEW

File: angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/rssfeed.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+   <ul class='widget-list'>
+      <li ng-repeat='item in data.responseData.feed.entries | filter:filterText'>
+        <a ng-href='{{item.link}}' target='_blank'>
+          <p class='bold'>{{item.title}}<span class='right' ng-if='config.showdate'>{{item.publishedDate}}</span></p>
+          <p ng-if='config.showsnippet'>{{item.contentSnippet}}</p>
+        </a>
+      </li>
+   </ul>
+</div>
+<a class='btn btn-default launch-app-button ng-scope' href='/portal/p/course-services'>Launch Full App</a>
```
Comment (apetro) [inline angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/rssfeed.html]: Looks like the launch target URL is being hard-coded in the template partial?

**Comment by keirserrie** [comment]
Whoa, very cool. Scroll bar seems like too much interface, would prefer just having the first 5 with extra padding. What do others think?

**Comment by apetro** [comment]
Knee-jerk : I like this a lot.

In the same vein as what @keirserrie is thinking about the scroll bar, more generally, it looks like this has several features:

 * scroll bar for many items
 * show item excerpt, or not
 * show item published date, or not

Those are probably all features we'll need sooner or later.  I wonder about simplifying them all away initially and adding them when they come up.

File: angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/rssfeed.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div ng-controller='RSSWidgetController as RSSWidgetCtrl' style='overflow-y: scroll; max-height: 210px;'>
+   <ul class='widget-list'>
+      <li ng-repeat='item in data.responseData.feed.entries | filter:filterText'>
+        <a ng-href='{{item.link}}' target='_blank'>
+          <p class='bold'>{{item.title}}<span class='right' ng-if='config.showdate'>{{item.publishedDate}}</span></p>
+          <p ng-if='config.showsnippet'>{{item.contentSnippet}}</p>
+        </a>
+      </li>
+   </ul>
```
Comment (apetro) [inline angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/rssfeed.html]: Zero news items empty state?  Error state?

Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/rssfeed.html]: True, no loading or error state at this time. Will do.

**Comment by jhanstra** [comment]
:+1:

**Comment by timlevett** [comment]
New screenshot

image

**Comment by timlevett** [comment]
Empty state now in place 

image

**Comment by keirserrie** [comment]
Looking good--would like to see in action, let's talk Tues.

**Comment by timlevett** [comment]
Sounds good @keirserrie

**Comment by timlevett** [comment]
Outcome from disscussion was: 
+ No function when you click the showing
+ Change text on bottom button to say "See all"

**Comment by timlevett** [comment]
Made changes from discussion.

screenshot 2015-08-18 at 1 49 22 pm

**Comment by vertein** [comment]
:+1:

**Pull request merged**